src/Resources/config/services.xml

```diff
@@ -4,171 +4,9 @@
-            <argument>%kernel.environment%</argument>
-            <tag name="frosh_tools.health_checker"/>
-        </service>
-
-        <service id="Frosh\Tools\Components\Health\Checker\PerformanceChecker\PhpSettingsChecker">
-            <tag name="frosh_tools.performance_checker"/>
-        </service>
-
-        <service id="Frosh\Tools\Components\Health\Checker\PerformanceChecker\AdminWorkerChecker">
-            <argument>%shopware.admin_worker.enable_admin_worker%</argument>
-            <tag name="frosh_tools.performance_checker"/>
-        </service>
-
-        <service id="Frosh\Tools\Components\Health\Checker\PerformanceChecker\MailOverQueueChecker">
-            <argument>%frosh_tools.mail_over_queue%</argument>
-            <tag name="frosh_tools.performance_checker"/>
-        </service>
-
-        <service id="Frosh\Tools\Components\Health\Checker\PerformanceChecker\IncrementStorageChecker">
-            <argument>%shopware.increment.user_activity.type%</argument>
-            <argument>%shopware.increment.message_queue.type%</argument>
-            <tag name="frosh_tools.performance_checker"/>
-        </service>
-
-        <service id="Frosh\Tools\Components\Health\Checker\PerformanceChecker\QueueConnectionChecker">
-            <argument>%frosh_tools.queue_connection%</argument>
-            <tag name="frosh_tools.performance_checker"/>
-        </service>
-
-        <service id="Frosh\Tools\Components\Health\Checker\PerformanceChecker\EsChecker">
-            <argument type="service" id="Frosh\Tools\Components\Elasticsearch\ElasticsearchManager"/>
-            <tag name="frosh_tools.performance_checker"/>
-        </service>
-
-        <service id="Frosh\Tools\Components\Health\Checker\PerformanceChecker\DisabledMailUpdatesChecker">
-            <argument type="service" id="parameter_bag"/>
-            <tag name="frosh_tools.performance_checker"/>
-        </service>
-
-        <service id="Frosh\Tools\Components\Health\Checker\PerformanceChecker\FixCacheIdSetChecker">
-            <argument>%kernel.shopware_version%</argument>
-            <tag name="frosh_tools.performance_checker"/>
-        </service>
-
-        <service id="Frosh\Tools\Components\Health\Checker\PerformanceChecker\LoggerLevelChecker">
-            <argument type="service" id="monolog.handler.business_event_handler_buffer"/>
-            <tag name="frosh_tools.performance_checker"/>
-        </service>
-
-        <service id="Frosh\Tools\Components\Health\Checker\PerformanceChecker\MySQL8Checker">
-            <argument type="service" id="Doctrine\DBAL\Connection"/>
-            <tag name="frosh_tools.performance_checker"/>
-        </service>
+        <defaults autowire="true" autoconfigure="true" />
+        <prototype namespace="Frosh\Tools\" resource="../../" exclude="../../{DependencyInjection,Resources,FroshTools.php}" />
 
         <service id="Frosh\Tools\Components\Messenger\TaskLoggingMiddleware">
```

Comment:
Can't we remove this too?

Reply:
Unfortunately, it is not working without those lines.